src/client/Client.cc

```diff
@@ -1775,7 +1775,6 @@ int Client::make_request(MetaRequest *request,
     r = request->get_abort_code();
     request->item.remove_myself();
     unregister_request(request);
-    put_request(request);
```

Comment:
Okay so after talking with Zheng about this, I think we should just keep the other commit which removes the redundant `Cap` argument.

Reply:
@renhwztetecs you can also add a comment here noting that the reference being put was acquired in `make_request`.

Reply:
I'm late
drop this! update